### DISCUSSION TITLE
Sensor failing when scheduler and worker restarts

### DISCUSSION THREAD

**Discussion by PrakharQ** [apache/airflow#28134] [Q&A]
Hi,  I have written a custom sensor which fails when the airflow scheduler and worker restarts if the sensor was in the polling state. Is there any configuration which can be set so it automatically starts polling after the restart is done?

**Comment by potiuk** [comment]
Did you check if it happens to another "standard" sensor? What's the difference in behaviour vs. any other sensor?
